scripts/resources/navbar.js

```diff
@@ -0,0 +1,13 @@
+var fs = require('fs');
+var ejs = require('ejs');
+const MARKDOWN_DIR = __dirname + '/../../templates';
+const TARGET = process.env.TARGET_API || 'ovp';
+const VERSION = process.env.OTT_API_VERSION || 'v5_1_0';
```

Comment:
Since the `VERSION` const here is only applicable to OTT, its name should be changed. Also `OTT_API_VERSION` should be documented in the README.